tests/agent/test_agent.py

```diff
@@ -175,7 +174,7 @@ def test_gx_agent_gets_env_vars_on_init(get_context, gx_agent_config):
     assert agent._config == gx_agent_config
 
 
-def test_gx_agent_invalid_token(monkeypatch):
+def test_gx_agent_invalid_token(monkeypatch, set_required_env_vars: None):
```

Comment:
Also here, setting other env vars before changing the token to "invalid" on the next line.